package/yast2-ruby-bindings.changes

```diff
@@ -1,4 +1,22 @@
 -------------------------------------------------------------------
+Wed Mar 22 15:45:50 UTC 2017 - jreidinger@suse.com
+
+- Add new y2base as ruby script which have several advantages
+  (bsc#1027181):
+-- Ruby is no longer embedded into YaST (instead we only use binary
+   extensions), so there is less chance of things breaking with
+   a new Ruby version
+-- show popup when signal is received and also allow YaST modules
+   to handle more gracefully when signal is raised
+-- better argument passing to y2base itself e.g.
+   y2base foo -S '(bar)' qt → y2base foo --arg bar qt
```

Comment:
Replace y2base with y2start also in the changes entry almost everywhere, except when quoting old usage:

... to y2start itself, e.g.
y2base foo -S '(bar)' '(baz)' qt → y2start foo --arg bar --arg baz qt